offlinevalidator/offlinevalidator.go

```diff
@@ -258,6 +269,71 @@ func checkNestedProperties(
 	}
 }
 
+func processNestedTemplates(properties map[string]interface{}, context *context.Context) {
+	if rawTemplateURL, ok := properties["TemplateURL"]; ok {
+		if templateURL, ok := rawTemplateURL.(string); ok {
+			err := validateNestedTemplate(templateURL, context)
+			if err != nil {
+				context.Logger.Error(err.Error())
+				os.Exit(1)
+			}
+		}
+	}
+}
+
+func validateNestedTemplate(templateURL string, context *context.Context) error {
+	mysession.InitializeSession(context)
+
+	tempfile, err := ioutil.TempFile("", "")
```

Comment:
Maybe we should configure where those temporary files go?
Of course with sane default (e.g. `/tmp` on UNIX-based systems).

Reply:
According to documentation, https://golang.org/pkg/io/ioutil/#TempFile  TempFile uses the default directory for temporary files if dir is the empty string.

Reply:
Yup, and that's great for a default - what I meant is the case when you want to contol where temporary files will land.